CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
 All notable changes to this project will be documented in this file.
 
+## [1.3.4] - 2021/11/09
```

Comment:
You can't have 2 changelogs with the same version. Please update the version following the semver rules. The version can be updated in NRAgent.xml:

```
    <!-- Properties -->
    <field id="version" type="string" value="1.3.4"/>
```

Since you added a new function and didn't modify the existing ones, yours is not a breaking change, and thus you have to update the minor version number: `1.4.0`

Reply:
My mistake, fixed.

Reply:
Amazing, now it's ready to be accepted. Thanks for your collaboration!